forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -194,41 +194,41 @@ open class DeviceBindingCallback : AbstractCallback, Binding {
             return
         }
 
+        //TODO We may need to delete other keys if we only want to maintain one keys on the device
+        //TODO However, we may want to have Application Pin as fallback, so for now, keep multiple keys.
+        var keyPair: KeyPair? = null
```

Comment:
It can be null, when deviceAuthenticator.generateKeys failed with Exception or timeout, the keyPair can be null, and would like to remove both the reference (from SharedPreference) and the actual Key if anything failed.